tests/integration/test_fim/test_registry/test_ignore/test_ignore_registry.py

```diff
@@ -94,17 +81,103 @@ def test_ignore_registry(key_string, key_object, tags_to_apply, get_configuratio
         Run test if matches with a configuration identifier, skip otherwise.
     """
     check_apply_test(tags_to_apply, get_configuration['tags'])
+    scheduled = get_configuration['metadata']['fim_mode'] == 'scheduled'
+    # Create registry
+    create_registry(registry_parser[root_key], os.path.join(registry, subkey), arch)
+    # Go ahead in time to let syscheck perform a new scan
+    check_time_travel(scheduled, monitor=wazuh_log_monitor)
+
+    if triggers_event:
+        event = wazuh_log_monitor.start(timeout=global_parameters.default_timeout,
+                                    callback=callback_detect_event,
+                                    error_message='Did not receive expected '
+                                                    '"Sending FIM event: ..." event',
+                                    accum_results=2).result()
```

Comment:
These lines are under-indented.
```suggestion
                                        callback=callback_detect_event,
                                        error_message='Did not receive expected '
                                                      '"Sending FIM event: ..." event',
                                        accum_results=2).result()
```